bedrock/firefox/templates/firefox/features/sync.html

```diff
@@ -0,0 +1,165 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button with context %}
+
+{% add_lang_files "firefox/shared" "firefox/sync" %}
+
+{% extends "firefox/features/base.html" %}
+
+{% block page_title %}
+  {% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+    {{ _('Use Bookmarks, Tabs and Passwords Across Devices | Firefox') }}
+  {% else %}
+    {{ _('Keep your Firefox in sync') }}
+  {% endif %}
+{% endblock %}
+
+{% block page_desc %}
+  {% if l10n_has_tag('firefox_title_meta_bug1345338') %}
+    {{ _('Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!') }}
+  {% endif %}
+{% endblock %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox-features-hub-detail' %}
+  {% stylesheet 'firefox-features-sync' %}
+{% endblock %}
+
+{% block body_id %}firefox-features-sync{% endblock %}
+
+{% block body_class %}
+  {{ super() }}
+  state-default
+{% endblock %}
+
+{% block features_header_image %}
+  {{ high_res_img('firefox/features/sync-hero.png', {'class': 'hero-image', 'alt': '', 'width': '700', 'height': '390'}) }}
+{% endblock %}
+
+{% block features_header_content %}
+  <h1><span>{{ _('Firefox Sync') }}</span></h1>
+{% endblock %}
+
+{% block features_list %}
+<section>
+  <div class="content">
+    <h2 class="section-title"><span>{{ _('Sync') }}</span></h2>
+
+    <div id="sync-intro-copy-cta-wrapper">
+      <div id="sync-intro-copy">
+        {# Warning Text #}
+        <div class="show-fx-29-30 show-fx-28-older warning">
```

Comment:
I know this is legacy stuff from the old page, but do we still need to cover detecting all these old FF versions separately? Could we just say `if FF < 31` then show an update message?